python/ELASTICACHE_REDIS_CLUSTER_AUTO_BACKUP_CHECK/ELASTICACHE_REDIS_CLUSTER_AUTO_BACKUP_CHECK.py

```diff
@@ -0,0 +1,444 @@
+# Copyright 2017-2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may
+# not use this file except in compliance with the License. A copy of the License is located at
+#
+#        http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for
+# the specific language governing permissions and limitations under the License.
+'''
+Rule Name:
+  ELASTICACHE_REDIS_CLUSTER_AUTOMATIC_BACKUP_CHECK
+
+Description:
+  Check whether the Amazon ElastiCache Redis clusters have automatic backup turned on. The rule is NON_COMPLIANT if the SnapshotRetentionLimit of an Amazon Elasticache Redis cluster is 0.
+
+Trigger:
+  Periodic
+
+Reports on:
+  AWS::ElastiCache::CacheCluster
+
+Rule Parameters:
+  snapshotRetentionPeriod
+   Optional

```

Comment:
Please follow the standard

Reply:
Modified the gherkin and parameter value